src/table/components/__tests__/TableTotals.spec.jsx

```diff
@@ -0,0 +1,215 @@
+import React from 'react';
+import { render, fireEvent } from '@testing-library/react';
+import TableTotals from '../TableTotals';
+import { getTotalPosition, getTotalInfo } from '../../../handle-data';
+import { TableContextProvider } from '../../context';
+import * as handleKeyPress from '../../utils/handle-key-press';
+import * as handleAccessibility from '../../utils/handle-accessibility';
+
+describe('<TableTotals />', () => {
+  const rootElement = {};
+  let tableData;
+  let theme;
+  let layout;
+  let selectionsAPI;
+  let keyboard;
+  let translator;
+  let numDims;
+
+  const renderTableTotals = (cellCoordMock) =>
+    render(
+      <TableContextProvider selectionsAPI={selectionsAPI} cellCoordMock={cellCoordMock}>
+        {(getTotalPosition(layout) === 'top' || getTotalPosition(layout) === 'bottom') && (
+          <TableTotals
+            rootElement={rootElement}
+            tableData={tableData}
+            theme={theme}
+            layout={layout}
+            keyboard={keyboard}
+            translator={translator}
+          />
+        )}
+      </TableContextProvider>
+    );
+
+  beforeEach(() => {
+    tableData = {
+      columns: [
+        { id: 1, isDim: true, dataColIdx: 0, totalInfo: 'Totals' },
+        { id: 2, isDim: false, dataColIdx: 1, totalInfo: '350' },
+      ],
+    };
+    theme = {
+      getColorPickerColor: () => {},
+      name: () => {},
+      getStyle: () => {},
+      table: { body: { borderColor: '' } },
+    };
+    layout = {
+      qHyperCube: {
+        qDimensionInfo: [
+          {
+            qFallbackTitle: 'Dim3',
+          },
+          {
+            qFallbackTitle: 'Expression1',
+          },
+        ],
+        qMeasureInfo: [
+          {
+            qFallbackTitle: 'Sum(Expression2)',
+          },
+          {
+            qFallbackTitle: 'Sum(Expression3)',
+          },
+        ],
+        qGrandTotalRow: [
+          {
+            qText: '350',
+          },
+          {
+            qText: '-',
+          },
+        ],
+      },
+      totals: {
+        show: false,
+        position: 'top',
+        label: 'Totals',
+      },
+    };
+    keyboard = {
+      enabled: false,
+    };
+    translator = { get: (s) => s };
+    selectionsAPI = {
+      isModal: () => false,
+    };
+  });
+
+  describe('Table total position:', () => {
```

Comment:
This should be the same as the function name
```suggestion
  describe('getTotalPosition', () => {
```